src/App.css

```diff
@@ -204,15 +205,71 @@ a:hover {
   bottom: 0;
   min-width: var(--sidebar-min-width);
   width: var(--sidebar-width);
+  padding: var(--sidebar-padding);
   background: var(--hero-purple);
+  overflow-x: hidden;
+  white-space: nowrap;
 }
 
+.sidebar-header {
+  color: white;
+  border-bottom: 1px solid white;
+  margin-bottom: 5px;
+  font-size: 24px;
+}
+
+.sub-module-progress-row-container {
+  cursor: pointer;
+  border-bottom: 1px solid white;
+  padding: 5px;
+  color: white;
+  display: flex;
+  align-items: center;
+}
+
+.progress-row-selected {
+  font-weight: bold;
+}
+
+.sub-module-title-container {
+  display: flex;
+  align-items: center;
+  width: 80%;
+  height: 48px;
+  padding: 5px;
+}
+
+.progress-circle-filled {
+  border: 2px solid white;
+  background-color: white;
+  border-radius: 50%;
+  width: 48px;
+  height: 48px;
+  min-width: 48px;
```

Comment:
Also include min-height: 48px, otherwise, the min-height will be auto and could result in the circle not being a circle at some points.

Reply:
![image](https://memecrunch.com/meme/HXNV/black-magic/image.jpg)